simple_install/install.bat.in

```diff
@@ -0,0 +1,180 @@
+@echo off
+
+@rem This script requires the user to install Python 3.9 or higher. All other
+@rem requirements are downloaded as needed.
+
+@rem change to the script's directory
+PUSHD "%~dp0"
+
+set "no_cache_dir=--no-cache-dir"
+if "%1" == "use-cache" (
+    set "no_cache_dir="
+)
+
+exit /b
+
+echo ***** Installing InvokeAI.. *****
+@rem Config
+
+@rem this should be changed to the tagged release!
+@rem set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/main.zip
+set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/lstein-simple-install.zip
+set INSTRUCTIONS='<insert URL here>'
```

Comment:
```suggestion
@rem ensure link to 3.10!! 3.11 is latest and users will download it
set INSTRUCTIONS='<insert URL here>'
```

Reply:
We should see whether python 3.11 works with InvokeAI. I tested on Windows with 3.10, and only 3.9 is supported on Ubunto 20.04 LTS.

Reply:
3.11 offers significant speedups *for Python code* - so probably downloads, checksums, *maybe* model loading (IDK the exact mechanism there).

Note that none of speedup work in 3.11 will help with anything connected to c, e.g. torch, numpy, numba... most of what we do 😆

Reply:
> 3.9 is supported on Ubunto 20.04 LTS

How have we not hit this before now (or have we and I never heard about it 🤔)?